### PR TITLE
Do not use forks in github

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
Do not use event.base_project in Github. As a GitHub app, we cannot authenticate to the fork projects.

Requires: https://github.com/packit-service/packit/pull/818

TODO:
- [x] Add tests for actual config loading. (We mock the getter in all tests.)

### CODE REVIEW

**Comment by lachmanfrantisek** [comment]
recheck

File: packit_service/config.py [hunk 1]
```diff
@@ -146,25 +144,28 @@ def get_service_config(cls) -> "ServiceConfig":
         return cls.service_config
 
 
-class PackageConfigGetterForGithub:
+class GeneralPackageConfigGetter:
```
Comment (TomasTomecek) [inline packit_service/config.py:147]: We should also have SergeantPackageConfigGetter

Comment (jpopelka) [reply packit_service/config.py:147]: aye aye captain ;-)

**Comment by TomasTomecek** [comment]
recheck

**Comment by lachmanfrantisek** [comment]
It has some problem with the centos-config loading.

Please, wait for https://github.com/packit-service/packit/pull/818

File: packit_service/config.py (hunk 1 above)
Comment (sakalosj) [reply packit_service/config.py:147]: i suggest BasePackgeConfigGetter, "Base" is used at least in ogr for similar, so to keep naming convention

Comment (csomh) [reply packit_service/config.py:147]: If it's a naming discussion, I must join!

Would calling it only `PackageConfigGetter` be good enough?

Comment (lachmanfrantisek) [reply packit_service/config.py:147]: Oh no. Naming...;(

My notes regardning the suggestion. (But I can pick any of these if you want.)

> `BasePac...`
- In OGR we use it between abstract class and the real implementation to share the code. Here, we are using this class as well.

> `PackageCo...` 
- This makes sense to me when we don't have any other implementation. (Like we have for Pagure now.)

Comment (lachmanfrantisek) [reply packit_service/config.py:147]: OK, as I am thinking about that. I'll merge those two classes to `PackageConfigGetter` and solve the difference in another way.

**Comment by lachmanfrantisek** [comment]
The PR in packit was merged and the code was a bit simplified.

PTAL

**Pull request merged**